lte/gateway/python/magma/pipelined/service_manager.py

```diff
@@ -508,11 +508,15 @@ class ServiceManager:
 
     def __init__(self, magma_service: MagmaService):
         self._magma_service = magma_service
-        if '5G_feature_set' not in magma_service.config:
-            self._5G_flag_enable = False
+        if 'enable5g_features' not in magma_service.config:
+            if 'enable5g_features' in self._magma_service.mconfig.enable5g_features:
```

Comment:
mconfig and yml config is already sorted out in main function, there is no need to redo it here.

Reply:
Yeah, updated this file as suggested.